src/app/games/abalone/abalone.component.ts

```diff
@@ -214,9 +214,11 @@ export class AbaloneComponent extends HexagonalGameComponent<AbaloneRules,
     private getArrowPath(single: boolean, direction: HexaDirection): { start: Coord, end: Coord, pointed: Coord } {
         let start: Coord = this.selecteds[0];
         let end: Coord = this.selecteds[this.selecteds.length - 1];
+        // If end and start are different, then single === false
```

Comment:
```
// si a alors b
if (b && !a)
```
c'est très confusant ! 
C'est quoi `single` ?

Reply:
Ok, alors je te propose de pas parler en code mais de parler en ce que ça veut dire
"if end and start are different" -> "if a group of piece is selected"